mkosi

```diff
@@ -2085,18 +2088,12 @@ def install_debian_or_ubuntu(args: CommandLineArguments,
                              *,
                              do_run_build_script: bool,
                              mirror: str) -> None:
-    repos = args.repositories or ["main"]
+    repos = set(args.repositories) or {"main"}
     # Ubuntu needs the 'universe' repo to install 'dracut'
-    if args.distribution == Distribution.ubuntu and args.bootable and 'universe' not in repos:
-        repos.append('universe')
+    if args.distribution == Distribution.ubuntu and args.bootable and args.with_unified_kernel_images:
+        repos.add('universe')
 
-    cmdline = ["debootstrap",
-               "--verbose",
-               "--variant=minbase",
-               "--include=systemd-sysv",
-               "--exclude=sysv-rc,initscripts,startpar,lsb-base,insserv",
```

Comment:
I think we should keep that around. They've been excluded since 2016 (I guess since the beginning).

Reply:
I checked, they aren't actually installed on debian unstable, I'll check with ubuntu as well just to be sure.

Reply:
Only lsb-base is installed by default on debian unstable and ubuntu focal which is just a shell functions library, the rest is not installed. I don't think we should exclude that in mkosi, it's harmless enough.

Reply:
Right, the list might contain remnants from earlier, when Debian did not default to systemd. I'll have a look later what debootstrap currently graces a system with.

Sure, `lsb-base` doesn't do much in itself, but it only provides function for system V init scripts, so we can (and probably should) skip it (like we're already doing). Unfortunately, more than enough Debian packages still pull them in anyways at least transitively because maintainers cannot be bothered with writing a service file, so if they're needed, they'll be there.

Reply:
I think this is one case where we don't gain much by deviating from the distro defaults. Let's leave it up to debian to fix this instead of doing it in mkosi. It might take up a little bit of extra space but that isn't bad enough to exclude it right?

Reply:
I guess we can add it to the roster, it won't hurt, but I don't think its necessary to any of our goals, i.e. it's an unnecessary piece of legacy code and it doesn't really interact with systemd (in the testing case) at all if nothing else pulls it back in.

Btw, I just noticed that this line of excluded has been there since the initial commit of mkosi. ;)